controllers/controllers_suite_test.go

```diff
@@ -371,3 +373,82 @@ func setupMachineCRDs() {
 		return ph.Patch(ctx, dummies.CSMachine1, patch.WithStatusObservedGeneration{})
 	}, timeout).Should(Succeed())
 }
+
+func setCSMachineOwnerCRD(owner *fakes.CloudStackMachineOwner, specReplicas, statusReplicas, statusReadyReplicas *int32, statusReady *bool) {
+	owner.Spec.Replicas = specReplicas
+	Ω(k8sClient.Create(ctx, owner)).Should(Succeed())
+	key := client.ObjectKey{Namespace: owner.Namespace, Name: owner.Name}
+	Eventually(func() error {
+		return k8sClient.Get(ctx, key, owner)
+	}, timeout).Should(BeNil())
+
+	Eventually(func() error {
+		owner.Status.Ready = statusReady
+		owner.Status.Replicas = statusReplicas
+		owner.Status.ReadyReplicas = statusReadyReplicas
+		return k8sClient.Status().Update(ctx, owner)
+	}, timeout).Should(BeNil())
+}
+
+// setCAPIMachineAndCSMachineCRDs creates a CAPI and CloudStack machine with an appropriate ownership ref between them.
+func setCAPIMachineAndCSMachineCRDs(CSMachine *infrav1.CloudStackMachine, CAPIMachine *clusterv1.Machine) {
```

Comment:
I sort of think these methods should be broken into another file. I'd like one file that just sets up the base objects an another to operate on their relationships. I'm fine with it as is if it's too much trouble, but I think that would be nice organizationally.